docs/Resources-Appendices/operators.md

```diff
@@ -39,22 +43,22 @@ z = x / ++x;
 Programs should not depend upon the order of evaluation of expressions, except as guaranteed by ANSI C for the following operators:
 
 ```
- 1.  a, b           comma operator (not the comma between arguments) 
+ 1.  a, b           comma operator (not the comma between arguments)
  2.  a && b         logical and
  3.  a || b         logical or
- 4.  a ? b : c      conditional 
+ 4.  a ? b : c      conditional
 ```
 
 All of these guarantee that expression `a` will be computed before expression `b` (or `c`).
 
 In addition, when a function-call takes place all arguments are evaluated before control transfers to the function.
 
 ```
```

Comment:
Based off the checklist provided in #18, 
- [ ] Make sure all code blocks use syntax highlighting
This could utilize a syntax highlight such as c, or c++.

Reply:
Same here, I don't think this has to be C in this case.